build_projects/dotnet-host-build/PrepareTargets.cs

```diff
@@ -118,25 +118,45 @@ public static BuildTargetResult GenerateVersions(BuildTargetContext c)
 
             var branchInfo = new BranchInfo(Dirs.RepoRoot);
 
+            // Read details from branchinfo.txt for our build configuration
+            int iMajor = int.Parse(branchInfo.Entries["MAJOR_VERSION"]);
+            int iMinor = int.Parse(branchInfo.Entries["MINOR_VERSION"]);
+            int iPatch = int.Parse(branchInfo.Entries["PATCH_VERSION"]);
+            string sReleaseSuffix = branchInfo.Entries["RELEASE_SUFFIX"];
+            bool fStabilizePackageVersion = bool.Parse(branchInfo.Entries["STABILIZE_PACKAGE_VERSION"]);
```

Comment:
I named it generally because we could consider using it outside of MNA if required.